src/microbe_stage/Microbe.Interior.cs

```diff
@@ -327,6 +337,45 @@ public void QueueEmitToxin(Compound toxinCompound)
         queuedToxinToEmit = toxinCompound;
     }
 
+    /// <summary>
+    ///   Tries to secrete slime if possible
+    /// </summary>
+    public void SecreteSlime(float delta)
```

Comment:
Is it possible handling this in SlimeJetComponent instead? Just like flagellum component and other ExternallyPositionedComponents, I think we should design organelle functionalities in a way that as much as the implementation should be put in their own respective components.

Reply:
If nothing else the slime secretion logic could be put in the organelle component to spare the poor Microbe class from having even more lines of code...

Reply:
I'm not sure how I can achieve that considering this method needs to know about all the slime jets in a cell, not just one.

Reply:
Then with this and the comment on `public List<SlimeJetComponent> SlimeJets { get; private set; } = new();` the approach used by toxin shooting is probably the right one.

Reply:
I gave this more thought. I think `SlimeJets` can stay so that `Microbe.SecreteSlime` can access all slime jets to do the actual secretion. The thing here is to make slime jets function more granularly, so the secretion logic, cooldown handling and movement speed boost all should be contained in `SlimeJetComponent`.

For example, adding speed boost can be done like this (taken from flagella code):
https://github.com/Revolutionary-Games/Thrive/blob/57f3233fccad0dc0110f30a144bbb4cea3b6759e/src/microbe_stage/organelle_components/MovementComponent.cs#L39-L42

And finally in `Microbe.SecreteSlime`, roughly what it'll look like:
```csharp
public void SecreteSlime(float delta)
{
        foreach (var jet in SlimeJets)
        {
                jet.Secrete();

               // Or if the cooldown should still account for all slime jets then `Secrete` can just return the amount secreted here.
               // totalSlimeSecreted += jet.Secrete();
        }

         // Cooldown handling...
}
```

Let me know if something is not clear.

Reply:
@Kasterisk I've added the closest approach I could manage. Due to things like the animation and sound effect, it turned out to be easier to keep a flag of whether each jet is active, which Microbe.Interior still controls. Then each jet component will eject mucilage and calculate its own force if it's set to active.